src/components/nav-header-panel/location-header-panel.styles.css

```diff
@@ -0,0 +1,12 @@
+.headerPanel {
```

Comment:
Why are we doing this? One of the reasons for going into carbon is that we leverage *their* CSS classes. We should write as little as possible custom CSS. Otherwise we derail from the pattern library.